utils/get_opt_externals

```diff
@@ -29,7 +29,9 @@ Possible choices for OPTEXT:
   RESTRICTED -- download optional components with stricter license than the
       LGPLv3 license of DFTB+ (typically GPL). The distribution of the compiled
       code will be restricted beyond the LGPL license and governed by the
-      strictest license of the included optional components (typically GPLv3).
+      strictest license of the included optional components (typically GPLv3)
+      or even forbidden in case of conflicting licenses (as with GPLv2). In
```

Comment:
Is this left over from the original ChIMES licensing decision, or are there other GPL2 components?

Reply:
Remaining from the original ChIMES license, deleted.